examples/cpp/CMakeLists.txt.cpp_example.in

```diff
@@ -24,8 +24,19 @@ find_package(@PRODUCT_NAME_NOSPACE@ REQUIRED COMPONENTS cpp)
 
 add_executable(${PROJECT_NAME} @ADD_EXAMPLE_SOURCE@)
 
-configure_file("startserver.sh" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
-configure_file("stopserver.sh" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
+if(CMAKE_GENERATOR MATCHES Win64*)
+  configure_file("startserver.ps1" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
```

Comment:
Why not just copy them all all the time?

Reply:
What are you going to do in Bash on Linux with a ps1 script? What are you going to do with a bash script in powershell? What will the client think when we present them with scripts that don't natively exist on their given platform? We would risk confusing the client. [Matt]

Are you thinking of Linux users that prefer PowershellCore or Windows users that use Ubuntu on Windows? [Mike]

Reply:
No I am saying that you conditional pointless for something so meaningless. Additionally the conditional is is matching the generator for Win64, there is no such generator. What if I use NMake on windows, this won't match.

Reply:
How is installing the only script that will intuitively and natively execute on that platform meaningless? I'm sure a *nix developer is going to see a ps1 script and go "Oh boy! An excuse to install powershell!"

As far as detecting windows, fair. What would be more appropriate? Win32 or Win64? Because we seem to use both, and I never know which is going to be defined, especially since we in no way support 32 bit Windows (yet Windows dev loves to hang onto the past).

Reply:
WINNER: Copy both types of scripts. (I want powershell everywhere.)

Good catch on the Win64. Switching to WIN32.

Reply:
Y’all are confusing generators with platforms. If I’m on windows platform you want dll’s and ps1 files. I can use many generators on Windows and most of them won’t have the word Win64 in them. Only the current VS generators do. VS 2019 generator does not. Use platform checks for platform stuff. If you’re doing generator checks you are probably not doing something right. Their use should be Baer very infrequent.

Reply:
My point on the scripts is that we have plenty of examples of installing bat and bash scripts together. The effort to distinguish which platform to copy which script is really not necessary. Especially when you consider the proliferation of bash on windows now too.

Reply:
This was already switched to use platform check (WIN32).